code/game/objects/items/devices/defib.dm

```diff
@@ -309,9 +309,9 @@
 
 //Checks for various conditions to see if the mob is revivable
 /obj/item/shockpaddles/proc/can_defib(mob/living/carbon/human/H) //This is checked before doing the defib operation
-	if((H.species.flags & NO_SCAN))
+	if((H.species.flags & NO_SCAN && H.species.reagent_tag != IS_SYNTHETIC)) //Synths and FBPs should now bypass the NO_SCAN requirement
 		return "buzzes: \"Unrecogized physiology. Operation aborted.\""
-	else if(H.isSynthetic() && !use_on_synthetic)
+	else if(H.isSynthetic() && !use_on_synthetic && !si_only) // The Advanced defibs will now work on both flesh and synth
```

Comment:
si_only is about who can use the defib, not if the defib is advanced or not.
```suggestion
	else if(H.isSynthetic() && !use_on_synthetic && !advanced_pads) // The Advanced defibs will now work on both flesh and synth
```

Reply:
Only the advanced defibs are si_only anyway.